tests/cli/test_cli_backfill.py

```diff
@@ -2217,13 +2218,17 @@ def test_complete_partitioned_backfill(
     @patch("google.cloud.bigquery.Client")
     @patch("subprocess.check_call")
     @patch("bigquery_etl.cli.backfill.deploy_table")
+    @patch("bigquery_etl.cli.backfill.Schema.from_query_file")
     def test_initiate_partitioned_backfill(
-        self, mock_deploy_table, check_call, mock_client, runner
+        self, mock_from_query_file, mock_deploy_table, check_call, mock_client, runner
     ):
+        Backfill_staging_table_name = (
```

Comment:
Why capitalize this?
```suggestion
        backfill_staging_table_name = (
```